test/CMakeLists.txt

```diff
@@ -209,4 +209,7 @@ add_subdirectory(wrapper)
 if(GPU_TARGETS MATCHES "gfx11")
     add_subdirectory(wmma_op)
 endif()
+if(GPU_TARGETS MATCHES "gfx942")
```

Comment:
I guess this is problematic because if one *also* targets other platforms, this will also get built and causing unsupported instructions error